app/app.go

```diff
@@ -646,6 +646,11 @@ func NewOKExChainApp(
 	app.SetGetTxFeeHandler(getTxFeeHandler())
 	app.SetEvmSysContractAddressHandler(NewEvmSysContractAddressHandler(app.EvmKeeper))
 	app.SetEvmWatcherCollector(app.EvmKeeper.Watcher.Collect)
+	if appconfig.GetOecConfig().GetDynamicGpMode() != types.MinimalGpMode {
```

Comment:
It may be crash, when set init MinimalGpMode, but set dynamic not MinimalGpMode

Reply:
Fixed. When MinimalGpMode is set at initialization, the app also initializes the "app.gpo". And when DeliverTx or ParallelTxs, all modes will collect the gasPrice and gasUsed of tx.